lib/wfsc/falco_wfsc_loop.m

```diff
@@ -288,6 +301,27 @@
         end
         hProgress = falco_plot_progress_testbed(hProgress, mp, Itr, out.InormHist_tb, Im_tb, DM1surf, DM2surf);
     else
-        hProgress = falco_plot_progress(hProgress, mp, Itr, out.InormHist, Im, DM1surf, DM2surf, ImSimOffaxis);
+        %hProgress = falco_plot_progress(hProgress, mp, Itr, out.InormHist, Im, DM1surf, DM2surf, ImSimOffaxis);

```

Comment:
@davidsmarx It looks like this comments out the default case for simulation and makes it OMC specific. Is that right? If so, please move it to a specific OMC simulation function with another if statement or case-switch statement within the simulation part of the outer if statement.